src/Symfony/Component/Console/Output/AnsiColorMode.php

```
@@ -78,7 +78,7 @@ private function convertFromRGB(int $r, int $g, int $b): int
 
     private function degradeHexColorToAnsi4(int $r, int $g, int $b): int
     {
-        return round($b / 255) << 2 | (round($g / 255) << 1) | round($r / 255);
+        return round($b / 255) << 2 | (round($g / 255) << 1) | (int) round($r / 255);
```

The PHP implicit casting is actually *safer* than an explicit int cast here. It would warn in case of precision loss while `(int)` silently succeeds. So -1 for this change